src/service/service.js

```diff
@@ -187,11 +191,18 @@ module.exports = class KahootService {
       const playerAnswer = trivia.questions[counter].answers.find(
         (answer) => answer.id === answerId
       );
+
       namespace.miniPodium.find((option) => option.option === playerAnswer.id).count++;
       if (playerAnswer.isCorrect) {
         socket.score += namespace.timer;
       }
-
+      const { description = '' } = trivia.questions[counter].answers
+        .find(({isCorrect}) => isCorrect)
+      socket.emit('answer-check', {
+        playerId: socket.playerId,
+        ...playerAnswer,
+        correctDescription: description
```

Comment:
Does that mean that answers are available on the client-side before the attendee can answer?

Reply:
no we don't have it
<img width="539" alt="Screenshot 2022-03-29 at 14 26 03" src="https://user-images.githubusercontent.com/14543603/160601240-f010cd20-a844-479a-92a9-70aa52000580.png">